src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -9,7 +9,7 @@
 UNKNOWN_REASON = "Unknown reason"
 NO_JOB_ERR = "No valid job ids"
 
-LOOKUP_TIMER_INTERVAL = 10
+LOOKUP_TIMER_INTERVAL = 5
```

Comment:
decrease lookup timer interval so frontend is a bit snappier

Reply:
That might take a little stress testing. This was more relevant for the older Execution Engine, but even with 10 seconds, there would occasionally be requests queueing up - and it could take a while to resolve them.

But I think it's fine. Ideally I'd like to see actual push requests from EE2 on job state change, but we're not there. :)